datafusion/core/src/physical_plan/projection.rs

```diff
@@ -51,6 +56,10 @@ pub struct ProjectionExec {
     schema: SchemaRef,
     /// The input plan
     input: Arc<dyn ExecutionPlan>,
+    /// The output ordering
+    output_ordering: Option<Vec<PhysicalSortExpr>>,
+    /// The alias map used to normalize out expressions like Partitioning and PhysicalSortExpr
+    alias_map: HashMap<Column, Vec<Column>>,
```

Comment:
Better to add comments to indicate what does the key & value stand for.

For my understanding, the key is the column in the input schema of this Projection operator. While the values are the columns in this output schema of this Projection operator.